tests/test_pytorch_dataloader_api.py

```diff
@@ -177,6 +177,61 @@ def test_no_duplicates_with_multiple_workers(self, tmpdir, input_shape, workers)
             assert len(unique_inputs) - 1 == batchindx
             assert len(unique_labels) - 1 == batchindx
 
+    def test_no_workers_info(self, tmpdir, input_shape, workers, mocker):
```

Comment:
What are we testing here exactly?

Reply:
It is basically the case where for some reason the `pytorch` library does not provide workers_info. We check for a `None` return for that and testing coverage was missing this statement.

Reply:
This case is the single worker case, so rename.

Reply:
Moreover, can't we test this within the fixture? Do we need a full extra test?